### PR TITLE
Version 2 - typescript, new API, new build process, lots of tests, etc.

### PR DESCRIPTION
**Pull request by briangonzalez**

### CODE REVIEW

**Comment by briangonzalez** [comment]
@AlfredJKwack Take a peek at this when you get a moment. Would love your feedback. My hope is to go live this week with it!

**Comment by briangonzalez** [comment]
I am going to go for it :)

File: README.md [hunk 1]
```diff
@@ -1,104 +1,76 @@
 # <img src="https://rawgithub.com/briangonzalez/rgbaster.js/master/demo/baster.svg" width=25 style="margin-right: 10px"> RGBaster
 
-A dead simple javascript library for extracting the dominant color(s) from an image.
+A dead simple, zero-dependency, promise-based javascript library for extracting the dominant color(s) from an image.
 
-### Usage
+> Version 2 was written from the ground up with a cleaner, more modern API, amore robust test suite, and is written in Typescript.
```
Comment (eltonmesquita) [inline README.md]: May I suggest a little improvement:
> Version 2 was written from the ground up with a clean and modern API, a robust test suite and is fully written in Typescript.

Comment (briangonzalez) [reply README.md]: Sounds good @eltonmesquita :)

Comment (briangonzalez) [reply README.md]: Updated

**Comment by AlfredJKwack** [comment]
> @AlfredJKwack Take a peek at this when you get a moment. Would love your feedback. My hope is to go live this week with it!

Ask an ye shall receive @briangonzalez : 1 patch and 3 issues opened.

**Pull request merged**